Model/Api.php

```diff
@@ -54,7 +54,7 @@ class Api
     protected $contactModel;
 
     /** @var ContactSource */
-    protected $contactSource;
+    public $contactSource;
```

Comment:
Please leave these protected, and add getters/setters if needed in onLeadPreSave and onLeadPostSave

Reply:
protected